src/components/ExampleDisplayComponent.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Box } from '@material-ui/core';
+import { UnControlled as ReactCodeMirror } from 'react-codemirror2';
+import { makeStyles } from '@material-ui/core/styles';
+import CodeMirror from 'codemirror';
+import '../style/CodeMirrorComponent.css';
+import 'codemirror/lib/codemirror.css';
+import 'codemirror/theme/material.css';
+import 'codemirror/addon/fold/foldgutter.css';
+require('codemirror/addon/mode/simple');
+require('codemirror/addon/edit/closebrackets');
+require('codemirror/addon/display/placeholder');
+require('codemirror/addon/comment/comment');
+require('codemirror/addon/fold/foldgutter');
+require('codemirror/addon/fold/brace-fold');
+require('codemirror/mode/xml/xml');
+require('codemirror/mode/javascript/javascript');
+
+// Define FSH syntax highlighting
+// Regular expressions from https://github.com/standardhealth/vscode-language-fsh/blob/master/syntaxes/fsh.tmLanguage.json
+CodeMirror.defineSimpleMode('fsh', {
```

Comment:
I'm noticing that the example window expands and contracts with what is selected, and that can get a little confusing, things jump around a little. I think we may be better off just allowing scroll bars for the example, and for the dropdown menu.

Reply:
Working on adding that now!

Reply:
Commits `0189a73` and `08b80ff` fix the height of the FSH Examples window and editor so that they don't bounce around so wildly. I'm having some problems adding separate scroll bars to the editor and file tree though. The dialog component that houses everything seems to override any scroll properties that I set in the child components in favor of the window-wide scroll that we see here. This isn't ideal so I'll continue working on it, but it may be a battle for another day.

Reply:
The fixed height feels much better to me than the jumping resizing from before, and the scroll appears when the editor needs to scroll vertically or when the file tree needs to scroll vertically, and the editor has it's own horizontal scroll bar. So if it does need to be a battle for another day, I think this is a huge improvement, and that battle can be fought later.